app/screens/Course/Course20.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import { View, Text } from 'react-native';
+import { exchangeSlice } from '../../redux/slices';
+import { HomeScreen } from '..';
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { Course } from './Course';
+import { styles } from './course.styles';
+import { text } from './text';
+
+export const Course20 = () => {
+  const data20 = '01.01.2020';
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch(
+      exchangeSlice.actions.getCourseStatistic({
+        payload: '20',
+        type: 'getCourseStatistic',
+      }),
+    );
+  }, [dispatch]);
+
+  const data20usd = useAppSelector(
+    state => state.course.course['2020'].exchangeRate[23].saleRateNB,
+  );
+
+  const data20eur = useAppSelector(
+    state => state.course.course['2020'].exchangeRate[8].saleRateNB,
+  );
```

Comment:
The same as previous screen
A lot of code duplication
If you need change something for ALL of these screens (because they are almost equal) - you need to do a lot of copypaste work - it isn't OK.